### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#30023]
#### VFS team name
eBenefits

#### VFS product name
Dependency Verification

#### Point of Contact/Reviewers
Angela Fowler (Accessibility)

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
Form elements should be labeled according to the data required.

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
When adding a dependent, the street address field is simply labeled "street," potentially causing the user to think that is all that is required.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved
